resources/views/pilots/notify-new-eta.blade.php

```diff
@@ -0,0 +1,10 @@
+@extends('_layouts.main')
+@section('page')
+    <div class="container">
+        <h1 class="fs-2 mb-4 font-display text-primary-emphasis">Notify New Entry Time</h1>
+        <p>
+            Some sort of text about when to use this form
```

Comment:
lorem ipsum strikes again

Reply:
Should be "Use this form to notify the controller when your entry time changes by xx minutes or more." Not sure what the actual requirement is.